test/ringpop-assert.js

```diff
@@ -910,8 +914,9 @@ module.exports = {
     waitForJoins: waitForJoins,
     waitForPingReqs: waitForPingReqs,
     waitForPing: waitForPing,
-    waitForEmptyPing: waitForEmptyPing,
-    // drainSUTDissemination: drainSUTDissemination,
+    // waitForEmptyPing: waitForEmptyPing,
+    waitForEmptyPing: fasterWaitForEmptyPing,
```

Comment:
If we really don't need `waitForEmptyPing`, we should properly rename the exported function. Since no outside projects depend on it, I think we can just do it.